.github/workflows/poetry-lock-command.yml

```
@@ -0,0 +1,144 @@
+        # Important that this is set so that CI checks are triggered again
+        # Without this we would be forever waiting on required checks to pass
+        token: ${{ secrets.GH_PAT_APPROVINGTON_OCTAVIA }}
+
+    - name: Checkout PR (${{ github.event.inputs.pr }})
+      uses: dawidd6/action-checkout-pr@v1
+      with:
+        pr: ${{ github.event.inputs.pr }}
+
+    - name: Get PR info
+      id: pr-info
+      run: |
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)